modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -30,6 +30,11 @@ endif::mapi[]
 * While the images are the same, the Azure Marketplace publisher is different depending on your region. If you are located in North America, specify `redhat` as the publisher. If you are located in EMEA, specify `redhat-limited` as the publisher.
 * The offer includes a `rh-ocp-worker` SKU and a `rh-ocp-worker-gen1` SKU. The `rh-ocp-worker` SKU represents a Hyper-V generation version 2 VM image. The default instance types used in {product-title} are version 2 compatible. If you are going to use an instance type that is only version 1 compatible, use the image associated with the `rh-ocp-worker-gen1` SKU. The `rh-ocp-worker-gen1` SKU represents a Hyper-V version 1 VM image.
 
+[IMPORTANT]
+====
+Installing images with the Azure marketplace is not supported on `arm64` clusters.
+====
```

Comment:
maybe say "clusters with arm64 instances" ? so when we have heterogeneous clusters it would be clear that it consists of multiple arch instances

Reply:
I like that wording better as well, thank you!